_commands/scripting/yes.md

```diff
@@ -13,12 +13,24 @@ y
 y
 ~~~
 
-<!--more-->
-
 ### Useful Options / Examples
 
-#### `yes custom_thing`
+#### `yes | rm *.txt`
+
+~~~ bash
+$ yes | rm *.txt
```

Comment:
Can you expand this example a bit to explain when using yes is useful / necessary (i.e. when does `rm` not just remove a file, when do you have to say yes to it?)